script.js

```diff
@@ -1,10 +1,26 @@
 
-var ApiKey = "h2gbHeoXuSGHYUHwer9Jy6S7mT5Sj8oP"
-var QueryURL = "https://app.ticketmaster.com/discovery/v2/events.json?city=atlanta&apikey=" + ApiKey
-$.ajax({
-    url: QueryURL,
-    method: "GET"
-}).then(function(response){
-    console.log(response)
-})
 
+
+
+
+function cityName(){
```

Comment:
This function will be executed only when the page is loaded or refreshed. Si if the user wants to search for a city again after searching once the event listener may not trigger. So the event listener should be outside the function.